common.gradle

```
@@ -0,0 +1,79 @@
+apply plugin: 'application'
+apply plugin: 'java-library'
+
+sourceCompatibility = 8
+targetCompatibility = 8
+
+repositories {
+    mavenCentral()
+}
```

```suggestion
repositories {
    if (project.hasProperty('useMavenLocal') && useMavenLocal == "True") {
       project.logger.lifecycle('Using Maven Local Repository')
       mavenLocal()
    }
    mavenCentral()
}
```
This allows the property `useMavenLocal` to be set to `True` if a user wants to run from a local Maven repository.  This can be set in the `gradle.properties` file (generally located in `~/.gradle` in Linux).